lib/setupmanagers/qemuhck/qemu_machine.rb

```diff
@@ -689,7 +690,7 @@ def run(scope, run_opts = nil)
         dump_commands
       else
         unless @configured
-          run_config_commands
+          run_config_commands unless @run_opts[:restored]
```

Comment:
Let's name the new option `configured` instead.
If it is named `restored`, this statement will mean: "if restored, the machine is already configured"; we already have such a logic in `HCKTest` so let the class contain the concern.
Theoretically, we can also instantiate `QemuMachine` without "restoring" in one session; whether restoration is happening or not does not matter for `QemuMachine`.